core/src/main/java/com/adobe/granite/translation/connector/bootstrap/core/impl/BootstrapTranslationServiceFactoryImpl.java

```diff
@@ -84,26 +93,41 @@ public TranslationService createTranslationService(TranslationMethod translation
 			throws TranslationException {
 		log.trace("BootstrapTranslationServiceFactoryImpl.createTranslationService");
 
-		BootstrapTranslationCloudConfig bootstrapCloudConfg = (BootstrapTranslationCloudConfig) cloudConfigUtil
-				.getCloudConfigObjectFromPath(BootstrapTranslationCloudConfig.class, cloudConfigPath);
 
-		String dummyConfigId = "";
-		String dummyServerUrl = "";
+		BootstrapTranslationCloudConfig bootstrapCloudConfg = null;
+    LiltApiClient liltApiClient = null;
+
+		String liltConfigId = "";
+		String liltServerUrl = "";
 		String previewPath = "";
 
+    try {
+      Map<String, Object> param = new HashMap<String, Object>();
+      param.put(ResourceResolverFactory.SUBSERVICE, "bootstrap-service");
+      ResourceResolver resourceResolver = resolverFactory.getServiceResourceResolver(param);
+      Resource res = resourceResolver.getResource(cloudConfigPath);
+      bootstrapCloudConfg = (BootstrapTranslationCloudConfig) cloudConfigUtil
+        .getCloudConfigObjectFromPath(res, BootstrapTranslationCloudConfig.class, cloudConfigPath);
+      if (res != null) {
+        bootstrapCloudConfg = new BootstrapTranslationCloudConfigImpl(res);
+      }
+    } catch (LoginException e) {
+      log.error("Error while resolving config resource {}", e);
```

Comment:
It looks like this results in the translation service still being created, but with `liltApiClient` being `null` -- is that the right failure mode?  It looks like it might cause crashes later on.